2019/05-28-godot-and-android/Game.gd

```diff
@@ -0,0 +1,19 @@
+extends Node
+
+onready var touches_label : Label = $Container/VBoxContainer/Touches
+
+var touch_count := 0
+
+
+func _ready() -> void:
+	update_touches_label()
+
+
+func _input(event: InputEvent) -> void:
+	if event is InputEventScreenTouch and event.is_pressed():
+		touch_count += 1
+		update_touches_label()
```

Comment:
That way you don't need the extra methods, update the label directly or use a signal.